source/stylesheets/refills/_navigation.scss

```diff
@@ -284,7 +284,7 @@ header.navigation {
       position: relative;
 
       input[type=search] {
-        @include box-sizing(border-box);
+        box-sizing: border-box;
```

Comment:
Properties should be ordered background, border, border-radius, box-sizing, color, font-size, font-style, margin, padding, width